shared-plugins/new-device-notification/new-device-notification.php

```diff
@@ -105,12 +105,17 @@ public function set_cookie() {
 			$parts = parse_url( home_url() );
 			setcookie( $this->cookie_name, $this->cookie_hash, $tenyrsfromnow, COOKIEPATH, $parts['host'], false, true );
 		}
+
+		// Is this a VIP Go mapped domain?
+		if ( defined( 'VIP_GO_ENV' ) ) {
```

Comment:
The value of `VIP_GO_ENV` [can be false](https://github.com/Automattic/vip-go-mu-plugins/blob/16a271f3a518e6a95df7d730ce844acc3b627781/000-vip-init.php#L3-L8), so the check just for it being defined is adequate. Thanks for checking.